src/test/kotlin/no/nav/syfo/dialogmote/api/v2/TildelDialogmoteApiV2Spek.kt

```diff
@@ -0,0 +1,239 @@
+                        database.connection.run { this.createNewDialogmoteWithReferences(newDialogmoteAnnenArbeidstaker) }
+
+                        with(
+                            handleRequest(HttpMethod.Get, urlMoterEnhet) {
+                                addHeader(HttpHeaders.Authorization, bearerHeader(veilederCallerToken))
+                            }
+                        ) {
+                            response.status() shouldBeEqualTo HttpStatusCode.OK
+                            val dialogmoteList = objectMapper.readValue<List<DialogmoteDTO>>(response.content!!)
+
+                            dialogmoteList.size shouldBeEqualTo 2
+                            dialogmoteList.any { dialogmoteDTO -> dialogmoteDTO.tildeltVeilederIdent == veilederCallerIdent } shouldBeEqualTo true
+                            dialogmoteList.any { dialogmoteDTO -> dialogmoteDTO.tildeltVeilederIdent == veilederIdentTildelesMoter } shouldBeEqualTo false
+
+                            createdDialogmoterUuids.addAll(dialogmoteList.map { UUID.fromString(it.uuid) })
+                        }
+
+                        with(
+                            handleRequest(HttpMethod.Post, urlTildelMote) {
+                                addHeader(HttpHeaders.Authorization, bearerHeader(veilederCallerToken))
+                                addHeader(HttpHeaders.ContentType, ContentType.Application.Json.toString())
+                                setBody(
+                                    objectMapper.writeValueAsString(
+                                        TildelDialogmoterDTO(
+                                            veilederIdent = veilederIdentTildelesMoter,
+                                            dialogmoteUuids = createdDialogmoterUuids
+                                        )
+                                    )
+                                )
+                            }
+                        ) {
+                            response.status() shouldBeEqualTo HttpStatusCode.OK
+                        }
+
+                        with(
+                            handleRequest(HttpMethod.Get, urlMoterEnhet) {
+                                addHeader(HttpHeaders.Authorization, bearerHeader(veilederCallerToken))
+                            }
+                        ) {
+                            response.status() shouldBeEqualTo HttpStatusCode.OK
+                            val dialogmoteList = objectMapper.readValue<List<DialogmoteDTO>>(response.content!!)
+
+                            dialogmoteList.size shouldBeEqualTo 2
+                            dialogmoteList.all { dialogmoteDTO -> dialogmoteDTO.tildeltVeilederIdent == veilederIdentTildelesMoter } shouldBeEqualTo true
+                            dialogmoteList.all { dialogmoteDTO -> dialogmoteDTO.tildeltVeilederIdent == veilederCallerIdent } shouldBeEqualTo false
+                        }
```

Comment:
Hvorfor gjøre et get-kall for å sjekke at riktig veileder ble tildelt? Det er jo post-kallet som testes; her gjør man seg bare avhengig av at masse annen kode fungerer for at denne testen skal bli riktig, i stedet for å bare sjekke hva som er i datbasen direkte 🤔

Reply:
Høres ut som du foreslår å sjekke i databasen direkte 😊 Vi kan godt endre til at vi gjør det